nodes/src/nodes/milvus/milvus.py

```
@@ -98,15 +102,19 @@ def __init__(self, provider: str, connConfig: Dict[str, Any], bag: Dict[str, Any
+            if profile != 'local':
+                # Init the store
+                if self.host.startswith('https:') or self.host.startswith('http:'):
+                    self.client = MilvusClient(uri=self.host, token=self.apikey, timeout=self.timeout)
+                else:
+                    self.client = MilvusClient(uri=f'https://{self.host}', token=self.apikey, timeout=self.timeout)
             else:
-                self.client = MilvusClient(uri=f'https://{self.host}', token=self.apikey, timeout=20)
-        else:
-            self.client = MilvusClient(uri=f'http://{self.host}:{self.port}', timeout=20)
+                self.client = MilvusClient(uri=f'http://{self.host}:{self.port}', timeout=self.timeout)
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)